setup.cfg

```diff
@@ -73,7 +73,7 @@ install_requires =
     cookiecutter >= 1.7.3  # dependency issues in older versions
     dataclasses; python_version<"3.7"
     enrich >= 1.2.5
-    Jinja2 >= 2.10.1
+    Jinja2 >= 2.11.3
```

Comment:
We should not cap the Jinja version here because Ansible users will expect things to work with 2.11 and 3.x. After all, Ansible works with those versions.

Reply:
fwiw, my remarks were about ``Jinja2 >= 2.11.3`` vs ``Jinja2 >= 3.0.0``.

Reply:
Ahh, OK. I misunderstood. Whenever someone talks about dependencies and semver, the upper bound is what comes to my mind.

Reply:
Probably an upper bound like `<4.0` would make sense but on the other hand experience told me that upper constraints should be added only when you have knowledge of something already breaking you. Otherwise is like lottery, in half of cases you prevent a bug, in the other half you introduce a time-bomb conflict which is not real and you endup having to move the bound up and make a hotfix.